packages/video_player/video_player/README.md

```diff
@@ -140,3 +140,10 @@ and so on.
 To learn about playback speed limitations, see the [`setPlaybackSpeed` method documentation](https://pub.dev/documentation/video_player/latest/video_player/VideoPlayerController/setPlaybackSpeed.html).
 
 Furthermore, see the example app for an example playback speed implementation.
+
+### Video view type
+
+You can set the video view type of your controller (instance of `VideoPlayerController`) during its creation by passing the `videoViewType` argument.  
+If set to `VideoViewType.platformView`, platform views will be used instead of texture view on supported platforms (Android and iOS).
+
+Using `VideoViewType.platformView` is not recommended on Android due to a known [issue][3] affecting platform views on Android.
```

Comment:
I've added a note about the Android platform views to this README as well cc: @stuartmorgan-g

Reply:
Let's leave the specifics to the Android package's README. Let's replace this paragraph with something more generic that will be true regardless of what happens with that issue:

> The relative performance of the different view types may vary by platform, and on some platforms the use of platform views may have correctness issues in certain circumstances due to limitations of Flutter's platform view system.